### DISCUSSION TITLE
Welcome to Payload-SDK Discussions!

### DISCUSSION THREAD

**Discussion by dji-dev** [dji-sdk/Payload-SDK#61] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eskemeth** [comment]
you got a payload that can bypass red restricted zone for the DJI mini 3?

**Comment by Harshit111001** [comment]
Are there any plans to release a ros package for the PSDK since its being recommended over OSDK for the latest models?
